operations/identitydocuments.md

```diff
@@ -0,0 +1,331 @@
+  "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+  "Client": "Sample Client 1.0.0",
+  "IdentityDocuments": [
+    {
+      "Id": "e8a72a69-c20b-4278-b699-ab0400a32ecc",
+      "Type": {
+        "Value": "Passport"
+      },
+      "Number": {
+        "Value": "123456789"
+      },
+      "ExpirationDate": {
+        "Value": "2040-11-20T00:00:00Z"
+      },
+      "IssuanceDate": {
+        "Value": "2020-11-20T00:00:00Z"
+      },
+      "IssuingCountryCode": {
+        "Value": "CZ"
+      },
+      "IssuingCity": {
+        "Value": "Prague"
+      },
+      "IdentityDocumentSupportNumber": {
+        "Value": "S-123456"
+      }
+    }
+  ],
+  "ChainId": "3fa85f64-5717-4562-b3fc-2c963f66afa6"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ChainId` | string | optional | Unique identifier of the chain. Required when using [Portfolio Access Tokens](../guidelines/multi-property.md), ignored otherwise. |
+| `IdentityDocuments` | array of [Identity document update parameters](identitydocuments.md#identity-document-update-parameters) | required, max 10 items | Identity documents to be updated. |
+
+#### Identity document update parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the document. |
+| `Type` | [Identity document type update value](identitydocuments.md#identity-document-type-update-value) | optional | Type of the document (or `null` if the number should not be updated). |
+| `Number` | [String update value](_objects.md#string-update-value) | optional | Number of the document (e.g. passport number or `null` if the number should not be updated). |
+| `ExpirationDate` | [String update value](_objects.md#string-update-value) | optional | Expiration date in ISO 8601 format (or `null` if the expiration date should not be updated). |
+| `IssuanceDate` | [String update value](_objects.md#string-update-value) | optional | Date of issuance in ISO 8601 format (or `null` if the issuance date should not be updated). |
+| `IssuingCountryCode` | [String update value](_objects.md#string-update-value) | optional | ISO 3166-1 code of the `Country` (or `null` if the issuing country code should not be updated). |
+| `IssuingCity` | [String update value](_objects.md#string-update-value) | optional | City where the document was issued (or `null` if the issuing city should not be updated). |
+| `IdentityDocumentSupportNumber` | [String update value](_objects.md#string-update-value) | optional | Identity document support number. Only required for Spanish identity cards in Spanish hotels. |
+
+#### Identity document type update value
```

Comment:
I think I understand why this has been done like this, but these update values are getting messy.

Reply:
Agreed, unfortunately we ran into some crappy limitations of how additional attributes on UpdateValue are being handled, so in addition to generator modifications, we will also have to adjust the schema generation.